### PR TITLE
Refs #31169 -- Added release note for parallel test running changes.

### PR DESCRIPTION
**Pull request by carltongibson**
Follow up to #15421, adding release note as per https://github.com/django/django/pull/15421#issuecomment-1098921367 @adamchainz

### CODE REVIEW

File: docs/releases/4.1.txt [hunk 1]
```diff
@@ -355,6 +355,10 @@ Tests
   :meth:`~.SimpleTestCase.assertFormsetError` now support passing a
   form/formset object directly.
 
+* The :class:`.DiscoverRunner` now supports running tests in parallel on
+  systems, such as macOS and Windows, where the ``spawn``
+  :mod:`multiprocessing` start method is used.
```
Comment (adamchainz) [inline docs/releases/4.1.txt:360]: A little word smithing...

```suggestion
* The :class:`.DiscoverRunner` now supports running tests in parallel on
  macOS, Windows, and any other systems where the default
  :mod:`multiprocessing` start method is ``spawn``.
```

Could also go as the first point in the section? Much bigger news than the others!

Comment (carltongibson) [reply docs/releases/4.1.txt]: Done. Thanks! 👍

**Review by adamchainz — APPROVED** [review]
Brilliant! 👍

**Comment by carltongibson** [comment]
Thanks Adam

**Pull request merged**